### ISSUE TITLE
Missing link in PDF version

### ISSUE THREAD

**Issue by CRZ** [MicrosoftDocs/microsoft-style-guide#400]
Greetings,

The "What's New" section in the PDF version of the guide is missing the most recent (6/20) update for Accessibility items.  (The web version of this page still says its last update was 6/6, by the way.)

Thanks,
CRZ

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9c33cc66-057c-fc38-363b-3eb9e09b131a
* Version Independent ID: f9680f41-fde0-77de-63dc-badc606d7273
* Content: [What's new? - Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/whats-new)
* Content Source: [styleguide/welcome/whats-new.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/main/styleguide/welcome/whats-new.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by S-LRaillard** [comment]
Hi @CRZ , this has been fixed.
